docs/guidance/identifiers.md

```diff
@@ -0,0 +1,40 @@
+# Project identifiers
+
+A project identifier is a globally unique identifier for an infrastructure project. Every project in OC4IDS has a project identifier in the `id` field.
+
+Project identifiers can be used to join up data published at different times or from different systems; for example, including a project identifier in contracting data makes it possible to join up data on the design, construction and supervision contracts within a single infrastructure project.
+
+## Project identifiers in contracting data
+
+A common need is to access data about the contracting processes related to an infrastructure project. When contracting systems use consistent identifiers to refer to infrastructure projects, this becomes possible. An example use case is automatically checking which projects have related contracting data, and then manually filtering projects for further scrutiny, monitoring or data collection.
+
+There are different approaches to including project identifiers in contracting data, with the best solution depending on the context of an implementation:
+
+* **Include a free-text field for project identifiers in procurement systems** and work with each group that enters data to make sure this is populated according to some defined pattern.
+
+  Free-text entry of project identifiers can lead to data quality issues; for example, two groups can accidentally choose to use the same identifier for different projects.
+
+  However, this approach allows some checks to be run; for example, checking that all the contracting processes over a certain value from a given agency have a project identifier, and that the identifier matches a pattern or a local identifier list.
+
+  This approach also allows data on several contracting processes relating to a single project to be joined up.
+
+
+* **Establish a national project register managed by a central agency and integrated into procurement systems.** In this model, officials entering procurement information would look up a project from the national register to include its identifier in contracting data, and would have to request its addition if it was not already in the register.
+
+  This approach allows validation ‘at source’ of the project identifier and would support checks on how many projects have no associated contracts, however it would require the availability of a central actor, the political will and the technical capacity to establish and maintain a register.
+
+## Globally unique identifiers
+
+The project identifier should be globally unique, this means that across data from all OC4IDS publishers, it should refer to one specific infrastructure project.
+
+To make the project identifier globally unique the OC4IDS helpdesk assigns a prefix which publishers must add to existing project identifiers, using the following identifier structure: `[project identifier prefix]-[existing project identifier]`
```

Comment:
Add comma after unique. Capitalize Helpdesk (also in admonition).

which -> that

It's not clear what an "existing project identifier" is (especially in contexts where there isn't a practice of assigning identifiers). A sentence (or more) should be added to relate this section to the previous section. We might want to use the phrase "local project identifier" in this section and the previous section, and to describe, in this section, that a prefix is added to a local project identifier to make it globally unique.

We might also want to describe, in the previous section, that a project identifier should be locally unique, in a similar way to the first paragraph in this section.

Reply:
I've tried to address your suggestions in https://github.com/open-contracting/infrastructure/pull/97/commits/acaa4b9fc84a4b1e1fba14ac3e30d85bda1a46f0

Happy for you to refine further by committing directly to the branch